scripts/testing/update_iso

```diff
@@ -22,6 +22,9 @@ PROJECT_DIR = os.path.dirname(
         )
     )
 
+# Virtual disk location
+VIRT_DISK = os.path.join(PROJECT_DIR, "tmp_virt_disk.qcow2")
```

Comment:
This would put the image in the root of the git repo, gitignore does not contain it, so it would be problematic for people who do `git add -u`

Should not the disk just be removed when the VM is deleted?

Reply:
In my cases it was not removed. Also we are using `qemu:///session` domain so the disc images will be nested in `~/.local/share/libvirt/images` which is not an obvious location.

I will update gitignore to include qcow2 files.

Reply:
how did you delete the VM?

Reply:
I was using `./scripts/testing/update_iso -b inst.sshd -k ../ks.cfg -v` which at the ends call `virt-install` with the newly created ISO image.

Reply:
I think we could also maybe use the python `tempfile` functionality to get a directory that will end up deleted once the virt-install call terminates ?

https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory

That should create the disk image in hopefully appropriate temporary location & then delete the image.

Reply:
I agree to what @M4rtinK says. We do the same in web ui tests.
https://github.com/rhinstaller/anaconda-webui/blob/main/test/anacondalib.py#L130

Reply:
Just beware, it will be default create the dir in /tmp that could be tricky if you don't have enough RAM.

Reply:
Yep, but it should be possible to change the dir where tempfile creates this to `/var/tmp` (located on normal storage) to be sure. :) Also nice we are already using this elsewhere. :)